src/app/views/band-albums.html

```diff
@@ -1,5 +1,28 @@
 <div class="band-detail-container">
-    <img src="https://i.scdn.co/image/66302ca80395b6be3600d5f0ef69db9e0c43f4f5" alt="">
-    <img src="https://i.scdn.co/image/e2b11f82ba8d5aecc6486fe28225db43a003caa1" alt="">
-    <img src="https://i.scdn.co/image/3a299e917835b053f53441280262f10829bd1c8e" alt="">
-</div>
\ No newline at end of file
+    <nav class="nav">
+        <img src="/app/images/spotify-icon.png" alt="" class="logo-left" ng-click=bandAlbumsCtrl.toIndex()>
+        <div class="column-search">
+            <input type="text" class="search-input" placeholder="Search for another artist" ng-model="bandAlbumsCtrl.search">
+            <i class="fa fa-search" aria-hidden="true" id="search-icon" ng-click="bandAlbumsCtrl.toResults()"></i>
+        </div>
+    </nav>
+    <div class="row">
+        <div class="inside-container">
+            <img src="{{bandAlbumsCtrl.artist.images[0].url}}" alt="" class="img-title">
+            <h1 class="title">{{bandAlbumsCtrl.artist.name}}</h1>
+            <h3 class="genre">{{bandAlbumsCtrl.artist.genres.join(", ")}}</h3>
+            <p class="links">
+                <a ng-click="bandAlbumsCtrl.toIndex()">HOME</a> > <a ng-click="bandAlbumsCtrl.backToResults()">ARTISTS</a> > {{bandAlbumsCtrl.artist.name}}
+            </p>
+        </div>
+    </div>
+    <div class="inside-container">
+        <div class="row">
+            <div class="item-album" ng-repeat="item in bandAlbumsCtrl.albums" ng-click="bandAlbumsCtrl.toAlbumDetail(item.id)">
```

Comment:
wrap the `img` and title with an `a` tag with the href with the path to the album